### ISSUE TITLE
Baseline feature

### ISSUE THREAD

**Issue by mirzazeyrek** [PHP-CS-Fixer/PHP-CS-Fixer#6451]
## Feature request

I was wondering if there are any plans for implementing a baseline feature like in phpstan or phpmd ? That would be great for legacy projects.

**Comment by stof** [comment]
Given that this tool is a fixer, not a linter reporting errors, I don't see how a baseline would work.

**Comment by Bilge** [comment]
In dry-run mode it is a linter.

**Comment by Wirone** [comment]
I am going to close this because:
- the feature request is too generic, it does not provide any information how baseline should work (on file level, file+fixer level, file+fixer+token index level?)
- it's nearly impossible that core team would work on it
- interest is rather low (author + 4 👍)
- Recently we introduced GitHub discussions with RFC and feature ideas sections, I believe we should start there and create issue with proper specification only if we're able to figure something out

There's no point for this issue to be open because in this form it does not help with implementation in any way.

**Comment by Bilge** [comment]
You hear him, guys. You didn't do enough emojis.

**Comment by Aeliot-Tm** [comment]
Hi, All!

There is just work around :upside_down_face:
https://github.com/Aeliot-Tm/php-cs-fixer-baseline

**Comment by sfaujour** [comment]
> Hi, @mirzazeyrek!
> 
> There is some kind of work around 🙃 https://github.com/Aeliot-Tm/php-cs-fixer-baseline

Top! How many emojis for not abandon this project after 6 months? 😄

**Comment by Bilge** [comment]
Minimum of 5 (unique) emojis per month to keep it alive.

**Comment by Wirone** [comment]
You guys are making fun about emojis, but in reality it's one of 2 _real_ available metrics showing people's interest in the issue/PR (the other one is comments' count). I did a full backlog review (300+ issues) and kept the ones I considered worth implementing even if it would require waiting some time to happen. But as a core team we just can't keep every idea open knowing we just don't have enough time to implement all of them. As I pointed out, there's a [better place](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/discussions/categories/feature-ideas) for feature ideas that anyone can pick up. Also, I really want to have _actionable_ issues, with clear scope and requirements, and this one simply was too generic to keep it. Thanks for understanding.

Also kudos to @Aeliot-Tm for providing userland workaround 🍻.
